.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,135 @@
+issue tracker is https://github.com/moira-alert/moira/issues. Issues that belongs for moira backend and web are placed here. 
+Issues that are related to python client and ansible role should be placed in issue tracker for related repo.
+
+We have a templates for two types of issues: Feature Request and a Bug. If you want to submit a bug please be patient 
+and fill all required fields. This will help us to define the problem. 
+
+Try to follow this advices to fill a great issue:
+
+* Check that issue tracker do not contain issues similar to your.
+* Use a clear and descriptive title for the issue to identify the suggestion.
+* Provide a step-by-step description of the suggested enhancement in as many details as possible.
+* Provide specific examples to demonstrate the steps. Include copy/pasteable snippets which you use in those examples, as Markdown code blocks.
+* Describe the current behavior and explain which behavior you expected to see instead and why.
+* Include screenshots and animated GIFs which help you demonstrate the steps to reproduce issue.
+* Explain why this enhancement would be useful to most Moira users.
+* Specify which version of Moira you're using.
+
+## Contributing a Code
+
+If you want to contribute a code you should choose an issue and comment that you want to work on it. If you cannot find 
+an issue for your contribution please fill an issue first. Please **DO NOT** work on code before you file an issue and comment
+about your wish to work on it.
+
+Here you can find issues that are good for first time contributors.
+
+* [Good First Issue](https://github.com/moira-alert/moira/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)
+* [Help Wanted](https://github.com/moira-alert/moira/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22)
+
+During implementation and submission of code follow style guides listed below.
+
+## Style Guides
+
+### Git Style Guide
+
+We are using [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) philosophy to submit the code.
+If you are not familiar with Conventional Commits it's highly recommended to read 
+[documentation](https://www.conventionalcommits.org/en/v1.0.0/) about it.
+
+In general the rules for commiting are following:
+
+* Commit message should have three blocks: title, body and footer divided with empty lines.
+* At least title should be specified.
+* Start first line with type of commit. You can find commit types below.
+* Place summary in the first line.
+* If applicable specify the scope of commit.
+* In body describe the changes you did. Write mostly **why** you did changes instead **what** you changed.
+* You can split a body to multiple paragraphs using blank lines.
+* In footer place an additional information about your commit. Specify issue number in footer with `Relates` or `Closes` label.
+
+Structure of commit:
+
+```
+<type>[optional scope]: <description>
+
+[optional body]
+
+[optional footer(s)]
```

Comment:
Provide an example of a good commit message?

Reply:
Provided